contents/docs/tutorials/cohorts.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Analyzing User Behavior with Cohorts
```

Comment:
Is: 'Analyzing the Behavior of DIfferent Groups of Users' clearer from a use case perspective?

Reply:
Sidebar width is a problem here:

<img width="600" alt="Screenshot 2020-11-02 at 08 33 08" src="https://user-images.githubusercontent.com/38760734/97846551-12e53880-1ce6-11eb-952b-421df4434e0e.png">

Reply:
Here are some titles that fit:

- Segmenting Users for Behavior Analysis
- Performing Analytics on User Segments

Reply:
In future we should make the title get cut off in the menu so I created #595 . For now, I'd leave it as is in this case